templates/shared/context.go

```diff
@@ -130,60 +132,60 @@ func Render(tpl *template.Template) func(ctx RuleContext) (string, error) {
 	}
 }
 
-func resolveRules(typ interface{ IsEmbed() bool }, rules *validate.FieldRules) (ruleType string, rule proto.Message, wrapped bool) {
+func resolveRules(typ interface{ IsEmbed() bool }, rules *validate.FieldRules) (ruleType string, rule proto.Message, messageRule proto.Message, wrapped bool) {
 	switch r := rules.GetType().(type) {
 	case *validate.FieldRules_Float:
-		return "float", r.Float, typ.IsEmbed()
+		ruleType, rule, messageRule, wrapped = "float", r.Float, rules.Message, typ.IsEmbed()
```

Comment:
do you notice that messageRule is always `rules.Message`. Any way to DRY this up?